modules/ROOT/pages/deploy-on-premises.adoc

```diff
@@ -309,6 +309,21 @@ mule -M-Dmule.deployment.parallel
 
 After performing these steps, Mule deploys in parallel all applications in the `/apps` directory.
 
+== Undeploying Applications
+
+You can undeploy Mule applications by deleting an app's anchor file only instead of deleting the application folder directly.
+
+Deleting only the app’s anchor file:
+
+* Prevents any interference from the hot-deployment layer and allows no room for concurrent conflicting actions.
+
+* Avoids potential application jar locking issues on some operation systems and allows for clean shutdown and undeployment.
```

Comment:
I think "jar" is a [JAR file](https://docs.oracle.com/javase/8/docs/technotes/guides/jar/jarGuide.html)

Reply:
Yes, it is! Modified it already.